cli/cmd/develop.go

```diff
@@ -47,35 +51,55 @@ type developerForgeProjectConfig struct {
 	templateName string
 }
 
-// newForgeProjectTemplate creates a new project using the forge template.
-func newForgeProjectTemplate(_ context.Context, cfg developerForgeProjectConfig) error {
-	// Check if forge is installed
-	if !checkForgeInstalled() {
-		// Forge is not installed, return an error with a suggestion.
-		return &cliError{
-			Msg:     "forge is not installed.",
-			Suggest: "You can install foundry by visiting https://github.com/foundry-rs/foundry",
+// newForgeProjectTemplate creates a new project by cloning a repository, checking out a specific commit if necessary, and initializing a new Git repository.
+func newForgeProjectTemplate(ctx context.Context, cfg developerForgeProjectConfig) error {
+	destinationPath, err := os.Getwd()
+	if err != nil {
+		return errors.Wrap(err, "failed to get current working directory")
+	}
+
+	// Clone the repository
+	cmd := exec.CommandContext(ctx, "git", "clone", "https://github.com/omni-network/"+cfg.templateName+".git", destinationPath)
```

Comment:
Since we are removing `.git` and need no history, do `--depth 1` in clone

Reply:
good one, added